climate.py

```diff
@@ -258,3 +258,64 @@ def get_upload_status(upload_id, token, api_key):
         return res.json()
     else:
         return None
+
+def get_scouting_observations(token, api_key, x_limit=100, x_next_token=None, occurredAfter=None, occurredBefore=None):
```

Comment:
at the python code level, we can still use good names and consistent style. we don't need to prefix these with `x_` nor do we need to mix in camelCase. similar to what we did for `get_fields`, we can call the token key `next_token`, and we can call the limit key just `limit`. occurred before/after keys should be `occurred_after` and `occurred_before`.

Reply:
Accepted.